interfaces/user.ts

```diff
@@ -59,6 +59,24 @@ export const UserRoleLabel: Record<UserRole, string> = {
   donor: "Donor",
 };
 
+export const RelationshipTypeConstants = <const>[
+  "Admin",
+  "Player",
+  "Parent to Player",
+  "Mentor to Player",
+  "Donor to Player",
+];
+
+// Use this to figure out what a user's role is from the user's viewerPermissions
+export type Role = typeof RelationshipTypeConstants[number];
```

Comment:
Are we able to use TS enums here instead, i.e.

```suggestion
export enum Role {
  Admin = 'admin',
  Player = 'player',
  ParentToPlayer = 'parent',
  MentorToPlayer = 'mentor',
  DonorToPlayer = 'donor
}
```

That way, we can use `Role.Admin` instead of "Admin", for example.

Reply:
thanks Ethan!!

Reply:
ah I realized that I actually don't need `RelationshipTypeConstants` and `Roles`, so I changed `RoleLabel` to be
```
export const RoleLabel: Record<string, string> = {
   Admin: "Admin",
   Player: "Player",
   "Parent to Player": "Parent",
   "Mentor to Player": "Mentor",
   "Donor to Player": "Donor",
 };
```